front-end/src/app/shared/scss/_global.scss

```diff
@@ -1,24 +1,21 @@
-
 @import '../../../../node_modules/bootstrap/scss/functions';
 @import '../../../../node_modules/bootstrap/scss/variables';
 @import '../../../../node_modules/bootstrap/scss/mixins';
 @import '../../../../node_modules/bootstrap/scss/grid';
-@import '../../../../node_modules/bootstrap/scss/dropdown';
-@import '../../../../node_modules/bootstrap/scss/tooltip';
-@import '../../../../node_modules/bootstrap/scss/card';
-@import '../../../../node_modules/bootstrap/scss/nav';
-@import '../../../../node_modules/bootstrap/scss/navbar';
-@import '../../../../node_modules/bootstrap/scss/modal';
-@import '../../../../node_modules/bootstrap/scss/utilities';
-@import '../../../../node_modules/bootstrap/scss/forms';
+// @import '../../../../node_modules/bootstrap/scss/dropdown';
```

Comment:
Can we remove these?

Reply:
There are a number of files that are hold overs to keep the login pages up and running until we migrate over to the new login system. These directories will be phased out after we have the new login setup:

- app/login
- app/shared/interceptors
- app/shared/services/AuthService
- app/shared/services/SessionService
- app/shared/scss